### PR TITLE
js bulby

### PR DESCRIPTION
**Pull request by aklepper20**
js-bulb | klepper

image

### CODE REVIEW

File: javascript-light-bulb/main.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+var lightOn = true;
+
+$bulbContainer.addEventListener('click', function () {
+  if (lightOn) {
+    $bulbContainer.className = 'bulb-container off';
+    $bulb.className = 'bulb off';
+  } else {
+    bulb.className = 'bulb';
+    bulbContainer.className = 'bulb-Container';
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:14]: Your "c" is capitalized in your class name 👀

File: javascript-light-bulb/main.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+var $bulb = document.querySelector('.bulb');
+var $bulbContainer = document.querySelector('.bulb-container');
+var bulb = 'bulb';
+var bulbContainer = 'bulb-container';
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:4]: These can be removed, they aren't being used

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js (hunk 2 above)
Comment (aklepper20) [reply javascript-light-bulb/main.js]: As I pushed it back up to github, js saw this as unused variables as an error and flagged them. I disabled that action in order to make the correct changes. Thank you!

File: javascript-light-bulb/main.js [hunk 3]
```diff
@@ -10,7 +8,9 @@ $bulbContainer.addEventListener('click', function () {
     $bulbContainer.className = 'bulb-container off';
     $bulb.className = 'bulb off';
   } else {
+    // eslint-disable-next-line no-undef
     bulb.className = 'bulb';
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:12]: You don't have a variable named `bulb`, you mean `$bulb`, which is the variable holding the DOM element

File: javascript-light-bulb/main.js [hunk 4]
```diff
@@ -10,7 +8,9 @@ $bulbContainer.addEventListener('click', function () {
...
-    bulbContainer.className = 'bulb-Container';
+    // eslint-disable-next-line no-undef
+    bulbContainer.className = 'bulb-container';
```
Comment (codingcodymiller) [inline javascript-light-bulb/main.js:14]: You don't have a variable named `bulbContainer`, you mean `$bulbContainer`, which is the variable holding the DOM element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: javascript-light-bulb/main.js (hunk 4 above)
Comment (aklepper20) [reply javascript-light-bulb/main.js]: image
 Noice

**Review by codingcodymiller — APPROVED** [review]
Nicely done!!

**Pull request merged**